documentation/docs/scenarios/xsa-deploy/Readme.md

```diff
@@ -1,13 +1,13 @@
 # Build and Deploy SAP Fiori Applications on SAP HANA XS Advanced
 
-Build an application based on SAPUI5 or SAP Fiori with Jenkins and deploy the build result into an SAP Cloud Platform account in the Neo environment.
+Build an application based on SAPUI5 or SAP Fiori with Jenkins and deploy the build result to SAP HANA XS Advanced.
```

Comment:
I just noticed that there might be a naming issue: The official name of SAP HANA XS Advanced is "SAP HANA extended application services, advanced model". Officially, we're not allowed to use "SAP HANA XS Advanced" - see [SAP Brand Tools](https://www.sapbrandtools.com/naming-center/#/requests/3436).